plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Tbh you were (and are) *only* blacklisting driver **released in 2015**, which is a bit different. 
Would it be so hard to transform the driver version number string to a 10 digit integer and blacklist stuff <1630023110 ?

As for downgrading, I'm pretty sure old dll mixing can solve 80% of people woes.